Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -147,7 +146,8 @@ import PayPalCheckout
               
                 PayPalCheckout.Checkout.start()
             case .failure(let error):
-                self?.apiClient.sendAnalyticsEvent("ios.paypal-native.create-order.failed")
+                self?.apiClient.sendAnalyticsEvent(BTPayPalNativeCheckoutAnalytics.orderCreationFailed)
```

Comment:
```suggestion
```
🧶 : Since the below `tokenizeFailed` event already gives us analytics coverage here.

Reply:
Sammy and Jax drew up a document for what events we want to keep. I will send you a link

Reply:
Oh that's cool if we done, we done!